### PR TITLE
Added Dark Matter Labs

### PR DESCRIPTION
**Pull request by elisapaka**

### CODE REVIEW

File: content/tbd/Dark Matter Labs.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  - Research
+cohere: true
+metadata_version: "1"
+---
+![[../assets/dark-matter-labs.png]] 
+
+> Dark Matter Labs (Dm) is working on what is needed to manifest transformations to our food, housing, land, material and nature systems towards a future of mutual thriving.  
+> 
+> In doing this, we analyse the shifts it requires in the underlying ‘dark matter’ - monetary, economic, governance, regulatory and policy systems - to make this possible, and we work with partners to demonstrate these alternatives in neighbourhoods, cities and bioregions. We share these insights openly for mutual learning.
+
```
Comment (catherinet1) [inline content/tbd/Dark Matter Labs.md:32]: Hi @elisapaka looks good -- only thing is to add links when quoting directly from website

File: content/tbd/Dark Matter Labs.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+## Key People
+
+## Theory of Change
+
+> We’re establishing toolkits and blueprints, pilots, and case studies, supporting communities and institutions with applications, digital products and civic technologies that challenge established thought and demonstrate that an alternative is possible.
```
Comment (catherinet1) [inline content/tbd/Dark Matter Labs.md:37]: this stuff i'd probably put above in the about/description section, as opposed to theory of change section, as it seems to me more about their activities/what they're doing as opposed to their theory or strategy or underlying principles of change.

**Review by catherinet1 — CHANGES_REQUESTED** [review]
main thing is links to the source of direct quotations

**Comment by rufuspollock** [comment]
@elisapaka does this replace this previous PR https://github.com/life-itself/ecosystem/pull/209 - if so let's close that old one with note that replaced by this.

**Pull request merged**